### ISSUE TITLE
Unable to use Windows Authentication in RHEL 8 (SEGV)

### ISSUE THREAD

**Issue by carofe82** [microsoft/azuredatastudio#19779]
Version: 1.37.0
Commit: d904740d93d7df76a0ba361f20e4351813b57645
Date: 2022-06-14T00:52:49.854Z
VS Code: 1.59.0
Electron: 13.6.6
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Linux x64 4.18.0-372.9.1.el8.x86_64

Steps to Reproduce:

1. Add SQL Server with AD, e.g. server,port
2. Select Windows Authentication
3. Connect
4. See SQL Tools Service client crashed (sometime message doesn't appear)

The systemd coredump reports 11 SEGV (segmentation fault) on `MicrosoftSqlToolsServiceLayer.` Command line in the coredump report looks like this:

```
resources/app/extensions/mssql/sqltoolsservice/Linux/4.0.1.1/MicrosoftSqlToolsServiceLayer --locale en --log-file /path/to/sqltools.log --tracing-level Critical --autoflush-log
```

The machine is joined to a domain with realm, and klist shows the token.

Does this issue occur when all extensions are disabled?: Yes

**Comment by cheenamalhotra** [comment]
Need to investigate further, and reproduce specifically with **kernel 4.18.0-372.9.1.el8.x86_64 (RHEL 8 x86_64)**, as its heavy backport on 4.18.0 kernel as well as extended x64 architecture.

**Comment by cheenamalhotra** [comment]
Also a duplicate of #18844.
Closing as will be addressed with **v1.39**.
